recipes/rosettasciio/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "rosettasciio" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rosettasciio-{{ version }}.tar.gz
+  sha256: 24c960f9a26441d826b96eda66007a958760b0dc965a11aaf4bc49884916ee05
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools >=49.0
+    - wheel
+    - cython
+    - pip
+  run:
+    - python
+    - dask-core >=2021.3.1
+    - python-dateutil
+    - h5py >=2.3
+    - imageio >=2.16
+    - numba >=0.52
+    - numpy >=1.20.0
+    - pint >=0.8
+    - python-box >=6,<7
+    - pyyaml
+    - scipy >=1.5.0
+    - sparse
+
+test:
+  imports:
+    - rsciio
+  commands:
+    - pip check
+    - pytest --pyargs rsciio
+  requires:
+    - pip
+    - pytest-xdist
+
+about:
+  home: https://hyperspy.org/rosettasciio
+  summary: Reading and writing scientific file formats
+  dev_url: https://github.com/hyperspy/rosettasciio
+  license: GPL-3.0-or-later
+  license_file: COPYING.txt
+
+extra:
+  recipe-maintainers:
+    - ericpre
+    - francisco-dlp
+    - jlaehne
```

Comment:
@francisco-dlp, @jlaehne, can you please confirm that you are happy to be maintainer of this recipe?

Reply:
Sure :-)

Reply:
Yes, I am happy to be maintainer of the recipe.